.github/workflows/testing.yml

```diff
@@ -0,0 +1,31 @@
+name: pytest
+
+# Triggers the workflow on push for all branches
+on: [push]
```

Comment:
The coveralls PR report does not get generated/updated if there is no testing work flow triggered on pull-request
That's why there was the testing-coveralls workflow before